File: lib/src/main/java/uk/iatom/rfm69/registers/values/Listen3.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum Listen3 implements IRegisterValue {
    

    // RegListen3),
    COEFRX_VALUE(0x20); // Default),;

    private byte val;
    
    Listen3(int val) {
        this.val = (byte) val;
    }

    @Override public byte val() {
        return val;
    }
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/Listen3.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class Listen3(`val`: Int): IRegisterValue { // RegListen3),
COEFRX_VALUE(0x20);


    // Default),;
    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
